RInAppMessaging/Classes/Protocols/Lockable.swift

```diff
@@ -42,15 +45,15 @@ internal class LockableObject<T>: LockableResource {
         if shouldWait {
             dispatchGroup.wait()
         }
-        lockCount += 1
+        lockCount = lockCount + 1
```

Comment:
How does this ensure thread safety?
Or is it because you've moved the decrement before the leave()?

Reply:
`@AtomicGetSet` doesn't work for incrementations. But actually this change doesn't help either. I'll have to fix that :)
Moving it before `leave()` might help though.